### DISCUSSION TITLE
JavaScript Bindings and ReferenceError: $ is not defined

### DISCUSSION THREAD

**Discussion by ecxs** [Budibase/budibase#9102] [Help]
Hi there,

After some searching, I figured out that `const temp = trigger.row.Department;` works, whereas const` temp = $("trigger.row.Department]");` does not, and returns the error 'ReferenceError: $ is not defined'.

`{
  "success": false,
  "response": "ReferenceError: $ is not defined"
}`

However, the documentation clearly states:

---
When using JavaScript instead of the handlebars, the syntax will be a little different, using dollar sign parenthesis. For example:

JavaScript

return $("Query 1.Customer.name")
To read more about JavaScript bindings, head over to the [Using JavaScript](https://docs.budibase.com/docs/javascript) guide.
---

What am I missing?

**Comment by shogunpurple** [comment]
seems like your binding isn't quite right in the JS one - there's a `]` in this binding = `$("trigger.row.Department]");`

Is this just a typo or is that the correct binding?

**Reply by ecxs** [reply]
Hi @shogunpurple sorry, this is indeed a typo (after having tried so many different formats) ... but $("trigger.row.Department") still doesn't work, whereas trigger.row.Department does ...
